src/d3a/models/strategy/load_hours.py

```diff
@@ -153,7 +153,6 @@ def _delete_past_state(self):
             del self.state.desired_energy_Wh[k]
 
     def _area_reconfigure_prices(self, **kwargs):
-        assert all(k in self.parameters for k in kwargs.keys())
```

Comment:
Why is this needed here ? The parameters should have exactly the same names as the incoming arguments from the live events

Reply:
Unfortunately, the update_event sents parameters like `uuid` and `name`, too which are not part of the `parameters` tuple. I agree that this was a quick and dirty fix and actually wanted to trigger the discussion here.
Why do we really need this assert here? We check only for the needed in the if clauses later anyway.

Reply:
You are right, better to not have this discussion here, and I understand the reason for removal. The assert is just a sanity check that validates that all the parameters passed are accepted by the strategy, but if parameters like uuid are passing through the update live event then feel free to remove this.